swan-lake/integration-tutorials/sending-a-message-to-a-service.md

```diff
@@ -137,17 +122,11 @@ Follow the instructions given in this section to develop the service.
         Doctor[]|http:ClientError resp = queryDoctorEP->/[category];
```

Comment:
Given the comments saying the code is kinda repeated, for this iteration, let's try removing the snippets from these descriptions and see.